### PR TITLE
Initial structure of the project

### PR DESCRIPTION
**Pull request by Klynger**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- Initial commit with Icon structure.
```
Comment (lucis) [inline CHANGELOG.md]: Icon structure?

File: react/components/Slider.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
```
Comment (lucis) [inline react/components/Slider.js:1]: Won't it be with Typescript?

File: .vtexignore [hunk 3]
```diff
@@ -0,0 +1,10 @@
+react/**/__tests__/**
+react/**/__mocks__/**
+react/node_modules/
+react/test-setup.js
```
Comment (lucis) [inline .vtexignore:4]: Will you include the test-setup?

**Review by lucis — APPROVED** [review]

File: .vtexignore (hunk 3 above)
Comment (Klynger) [reply .vtexignore:4]: For now I won't, but I thought that would be better put it already in here

File: react/components/Slider.js (hunk 2 above)
Comment (Klynger) [reply react/components/Slider.js:1]: It will be just JS for now

**Review by lucasecdb — APPROVED** [review]

**Pull request merged**